updateEBSVolumes.sh

```diff
@@ -54,7 +50,6 @@ do
   aws ec2 create-tags --tags Key=$namespaceKey,Value=$dfOfferingNamespace --resources $volumeID --region $region --profile migration
   
   #Update the volume type to gp3
-  pvName="${pv%.*}"
   claimName=$(kubectl get pv ${pvName} -ojson | jq -r '.spec .claimRef .name')
```

Comment:
I think we need to change here ${pvName} -> $pvName. Not sure if this works or not

Reply:
at line 53?, let's verify it after creating a cluster